docs/source/keywords.rst

```diff
@@ -347,91 +347,102 @@ Architectures
 CPU architectures
 -----------------
 
-.. list-table::
-    :widths: 25 65 10
-    :header-rows: 1
-    :align: left
-
-    * -
-      - Description/info
-      - Default
-
-    * * ``Kokkos_ARCH_NATIVE``
-      * Optimize for the local CPU architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_A64FX``
-      * Optimize for ARMv8.2 with SVE Support
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_AMDAVX``
-      * Optimize for AMDAVX architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_ARMV80``
-      * Optimize for ARMV80 architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_ARMV81``
-      * Optimize for ARMV81 architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_ARMV8_THUNDERX``
-      * Optimize for ARMV8_THUNDERX architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_ARMV8_THUNDERX2``
-      * Optimize for the ARMV8_THUNDERX2 architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_BDW``
-      * Optimize for BDW architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_HSW``
-      * Optimize for HSW architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_KNC``
-      * Optimize for KNC architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_KNL``
-      * Optimize for KNL architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_POWER8``
-      * Optimize for POWER8 architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_POWER9``
-      * Optimize for POWER9 architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_SKX``
-      * Optimize for SKX architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_SNB``
-      * Optimize for SNB architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_SPR``
-      * Optimize for Sapphire Rapids architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_ZEN``
-      * Optimize for Zen architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_ZEN2``
-      * Optimize for Zen2 architecture
-      * ``OFF``
-
-    * * ``Kokkos_ARCH_ZEN3``
-      * Optimize for Zen3 architecture
-      * ``OFF``
+Kokkos does not automatically add compiler flags to optimize for a specific CPU architecture.
+When compiling on the machine that the code also runs on, the easiest way to get the CPU code optimized is using ``Kokkos_ARCH_NATIVE=ON`` when configuring.
```

Comment:
I think we should consider placing that single option in a table to make it stand out.  I think it will be overlooked otherwise.

Reply:
done in https://github.com/kokkos/kokkos-core-wiki/pull/602/commits/9b757ae533b23dee75600b4ba1a1d9baa3f8f658